libs/ui-components/src/hooks/useDeviceSpecSystemInfo.tsx

```
@@ -64,11 +66,18 @@ export const useEnrollmentRequestSystemInfo = (
       return !excludedKnownProps.includes(infoKey) && systemInfo[infoKey];
     })
     .map(([infoKey, infoTitle]) => {
-      let infoDataValue = systemInfo[infoKey];
+      let infoDataValue: string;
 
-      if (infoKey === 'distroName') {
-        const version = 'distroVersion' in systemInfo ? ` ${systemInfo['distroVersion']}` : '';
-        infoDataValue = `${infoDataValue}${version}`;
+      switch (infoKey) {
+        case 'distroName':
+          const version = 'distroVersion' in systemInfo ? ` ${systemInfo['distroVersion']}` : '';
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)